codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttConnectPayload.java

```diff
@@ -59,6 +78,10 @@ public String userName() {
     }
 
     public String password() {
```

Comment:
Please deprecate this method and refer to the `passwordInBytes ` in the `@deprecated` javadocs.